### DISCUSSION TITLE
[SOLVED] Trying to localize the feed, getting multiple errors in translate-feed.py

### DISCUSSION THREAD

**Discussion by Nikodemsky** [martinrotter/rssguard#701] [General]
Hi,

i'm trying to get translated version of feed from japanese to english:
https://nitter.42l.fr/comic_natalie/rss

the thing is i'm getting multiple errors while running the script(means, when clicking on "fetch it now") and i'm not sure what i should do with it.

Notes:
* Got latest python
* Got all the modules installed
* Got all the dependecies installed

Any help would be much appreciated!

![python1](https://user-images.githubusercontent.com/14090596/165196320-0eb03e4a-1570-4807-b1f9-550053074def.png)

**Comment by martinrotter** [comment]
So you used the "official" script from here?

https://github.com/martinrotter/rssguard/blob/master/resources/scripts/scrapers/translate-feed.py

Will test it.

**Comment by martinrotter** [comment]
OK:
1. Do not use `#` as separator if you use RSS Guard 4.2.0 or newer, see this warning from documentation:
> As of RSS Guard 4.2.0, you cannot separate your arguments with #. If your argument contains spaces, then enclose it with DOUBLE quotes, for example "my argument". DO NOT use SINGLE quotes to do that.

2. Python 3.10+ obsoletes some functions. I therefore update the scraper - 2eb016c89. Should work now. See screens below.

![image](https://user-images.githubusercontent.com/1255302/165232217-4e950a76-258f-456a-bf81-9d6774bfe4ec.png)

![image](https://user-images.githubusercontent.com/1255302/165232283-f330576b-33fe-4702-9f06-1424b3d81ea7.png)
